### DISCUSSION TITLE
Build failed :| Scheme "staging" not found from repository! Please reconfigure your project.

### DISCUSSION THREAD

**Discussion by salmanclustox** [codemagic-ci-cd/codemagic-docs#1859] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

When i am trying to run my workflow for flutter ios build having multiple flavor code magic is giving this error on Installing dependencies task

Build failed :| Scheme "staging" not found from repository! Please reconfigure your project.

### Expected results

It should build the ipa using that scheme i used it before too in other projects it worked

### Actual results

Build failed :| Scheme "staging" not found from repository! Please reconfigure your project.

### Build id (optional)

64a8089076d9ef65200a52ad

**Comment by icarusdust** [comment, marked as answer]
Hi @salmanclustox, it is usually about either a typo in the scheme name or it needs to be built for the project's workspace and that can be configured in Xcode. Though, checking the latest logs and can see that you have already fixed it. Please confirm, so that we can close this thread.

**Reply by icarusdust** [reply]
Also, worth mentioning that your team is on a paid plan which means that you have access to the chat widget, please use that widget next time for quicker responses.
